src/sagemaker/experiments/run.py

```diff
@@ -633,7 +633,10 @@ def _extract_run_name_from_tc_name(trial_component_name: str, experiment_name: s
         Returns:
             str: The name of the Run object supplied by a user.
         """
-        return trial_component_name.replace("{}{}".format(experiment_name, DELIMITER), "", 1)
+        # TODO: we should revert the lower casting once backend fix reaches prod
+        return trial_component_name.replace(
+            "{}{}".format(experiment_name.lower(), DELIMITER), "", 1
+        )
```

Comment:
Good catch! So I assume this line would fix the issue of second run created? And will fix the list_runs issue https://github.com/aws/sagemaker-python-sdk/issues/3673?

Can we add a unit test which does list_runs twice in case to validate in case any future changes break it again?

Reply:
sure, will do